cockroachcloud/export-logs.md

```diff
@@ -48,7 +48,7 @@ When written to your chosen cloud log sink, logs have the following name format:
 
 Where:
 
-- `{log-name}` is a string of your choosing as you configure log export. For AWS CloudWatch, this is the [log group](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Working-with-log-groups-and-streams.html#Create-Log-Group) you create as part of enabling log export. For GCP Cloud Logging, this is the `log_name` you choose during configuration. See the [Enable log export](#enable-log-export) instructions specific to your cloud provider for more information.
+- `{log-name}` is a string of your choosing as you configure log export. For AWS CloudWatch, this is the [CloudWatch log group](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Working-with-log-groups-and-streams.html#Create-Log-Group). For GCP Cloud Logging, this is the `log_name` you choose during configuration. See the [Enable log export](#enable-log-export) instructions specific to your cloud provider for more information.
```

Comment:
This bullet point has become a bit confusing. I think it should instead say something like:

`{log-name}` is a string if your choosing as you configure log export. This name will correspond to an automatically created CloudWatch log group in AWS CloudWatch. In GCP Cloud Logging this string will be prepended to all log names sent from your cluster. etc etc.